presto-hive/src/main/java/io/prestosql/plugin/hive/BackgroundHiveSplitLoader.java

```diff
@@ -379,13 +389,48 @@ private void invokeNoMoreSplitsIfNecessary()
 
         // Bucketed partitions are fully loaded immediately since all files must be loaded to determine the file to bucket mapping
         if (tableBucketInfo.isPresent()) {
+            if (AcidUtils.isTransactionalTable(table.getParameters())) {
+                throw new PrestoException(NOT_SUPPORTED, "Presto cannot read bucketted hive transactional tables");
```

Comment:
IMO this is very important, given that transactional tables _are_ bucketed, before Hive 3.


(also, `bucketted` - stuttered `t`)

Reply:
Agreed, as mentioned in Insert Only support PR: this is one of the next things we will work on.

Reply:
I hoped you had this ready. Understood. I posted https://github.com/prestosql/presto/pull/1591.